sdk/monitor/monitor-opentelemetry-exporter/src/platform/nodejs/baseSender.ts

```diff
@@ -119,23 +125,29 @@ export abstract class BaseSender {
             });
           }
           if (filteredEnvelopes.length > 0) {
-            this.networkStatsbeatMetrics?.countRetry(statusCode);
+            if (!this.isStatsbeatSender) {
+              this.networkStatsbeatMetrics?.countRetry(statusCode);
+            }
             // calls resultCallback(ExportResult) based on result of persister.push
             return await this.persist(filteredEnvelopes);
           }
           // Failed -- not retriable
-          this.networkStatsbeatMetrics?.countFailure(duration, statusCode);
+          if (!this.isStatsbeatSender) {
+            this.networkStatsbeatMetrics?.countFailure(duration, statusCode);
+          }
           return {
             code: ExportResultCode.FAILED,
           };
         } else {
           // calls resultCallback(ExportResult) based on result of persister.push
-          this.networkStatsbeatMetrics?.countRetry(statusCode);
+          if (!this.isStatsbeatSender) {
+            this.networkStatsbeatMetrics?.countRetry(statusCode);
+          }
           return await this.persist(envelopes);
         }
       } else {
         // Failed -- not retriable
-        if (this.networkStatsbeatMetrics) {
+        if (this.networkStatsbeatMetrics && !this.isStatsbeatSender) {
```

Comment:
Is this another bug? are we not supposed to send retriable network statbeat data? if it is let's mention that in the changelog as well

Reply:
I've updated the singleton implementations to be consistent between network & long interval + ensured that the instance is a static attribute on the class for debugging ease.